apps/dnn-regression/train_dnn.py

```diff
@@ -71,10 +71,10 @@ def train(model_dir, training_pandas_data, test_pandas_data, label_col, feat_col
 
     print("Test RMSE:", test_rmse)
 
-    mlflow.log_param("Number of data points", len(training_pandas_data[label_col].values))
+    mlflow.log_param("num_train_points", len(training_pandas_data[label_col].values))
```

Comment:
Actually, could we log this further up? i.e. before we do any training logic just so we have this information even if the run fails midway through. Also would be good to log this info in the other apps for consistency